src/linker/Linker.Steps/UnreachableBlocksOptimizer.cs

```diff
@@ -537,6 +537,8 @@ struct BodyReducer
 			// 
 			List<int>? conditionInstrsToRemove;
 
+			List<(int, Instruction)>? conditionInstrsToReplace;
```

Comment:
Add a comment describing what is this for - I know it's mentioned below, but I think it's better to have it in one place.